sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -380,6 +381,41 @@ public String getCheckedOptionFromHorizontalOptionGroup(By options) {
     return baseSteps.getDriver().findElement(options).findElement(SELECTED_RADIO_BUTTON).getText();
   }
 
+  public String getOptionFromHorizontalGroupOrNullIfNothingChecked(By options) {
+    waitUntilIdentifiedElementIsPresent(options);
+    if (baseSteps.getDriver().findElement(options).findElements(SELECTED_RADIO_BUTTON).size()
+        == 0) {
+      return null;
+    } else {
+      return getCheckedOptionFromHorizontalOptionGroup(options);
+    }
+  }
```

Comment:
I think this method can be deleted, because if we checked something before than means next time we are checking the element is checked.

Reply:
please consider that that method is providing a null return when you check a group which could have selected radio buttons.

when you are creating a contact, you fill a form. on the edit contact page, we check that some of the groups are not selected. that's the purpose of those checks.

e.g. you have to check that after creating a a contact diabets GROUPS is not selected with YES or NO.